src/modules/customers/components/detail/sidebar/BasicInfo.js

```diff
@@ -0,0 +1,182 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Sidebar } from 'modules/layout/components';
+import { SidebarContent, QuickButton } from 'modules/layout/styles';
+import { AvatarWrapper } from 'modules/activityLogs/styles';
+import {
+  Button,
+  Icon,
+  FormControl,
+  FormGroup,
+  NameCard
+} from 'modules/common/components';
+import { Alert } from 'modules/common/utils';
+import {
+  AboutList,
+  Aboutvalues,
+  NameWrapper,
+  ButtonWrapper,
+  AboutWrapper
+} from './styles';
+
+const propTypes = {
+  customer: PropTypes.object.isRequired,
+  save: PropTypes.func.isRequired
+};
+
+class BasicInfo extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.defaultBasicinfos = {
+      ...(props.customer || {})
+    };
+
+    this.state = {
+      editing: false,
+      ...this.defaultBasicinfos
```

Comment:
Just spread only nessesary fields.

this.state = {
 firstName: customer.firstName,
 lastName:  customer.lastName,
...
}